<html>

<head>
    <!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-21895818-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];

        function gtag() {
            dataLayer.push(arguments);
        }
        gtag('js', new Date());

        gtag('config', 'UA-21895818-1');
    </script>
    <title>ManChon (Kevin) U, PhD</title>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Expires" content="0" />
    <meta http-equiv="Classification" content="Science, Education, Academics, Computer, Machine Learning, Artificial Intelligence, Hospitality" />
    <meta name="keywords" content="ManChonU, ManChon U, Man Chon U, U Man Chon, Kevin, Dr. U" />
    <meta name="description" content="Homepage of Man Chon (Kevin) U, Ph.D." />
    <meta name="rating" content="General" />
    <meta name="MS.LOCALE" content="EN-US" />
    <meta name="CATEGORY" content="Science, Education, Academics, Computer, Machine Learning, Artificial Intelligence, Hospitality" />
    <meta name="robots" content="index,follow" />
    <meta name="URL" content="https://manchonu.github.io/" />
    <link rel="stylesheet" type="text/css" href="talent.css" />
</head>

<body>
    <Div align=center>
        <table width=1024 border="0" cellspacing="0" cellpadding="0">
            <tr class="header" height="80">
                <td colspan="2" background="pic/header.gif" align="left">
                    <a href="index.html" target="_self"><img border="0" src="pic/logo.gif"></a>
                </td>
            </tr>
            <tr class="headerTag" height="30">
                <td colspan="2" background="pic/headerMsg.gif">
                    <Div class="left white t12 tb">
                     CX Expert | Driving Growth Through Digital Transformation & Data Science
                    </Div>
                </td>
            </tr>
            <tr class="headerTag" height="35">
                <td colspan="2" background="pic/tag.gif">
                    <Div class="left white">
                        <a class="aWhite" href="index.html">HOME</a> &#8226;
                        <a class="aWhite" href="cv.html">RESUME</a> &#8226;
                        <a class="aWhite" href="life.html">LIFE JOURNEY</a> &#8226;
                        <a class="aWhite" href="faq.html">FAQ</a> &#8226;
                        <a class="aWhite" href="research.html">PHD RESEARCH</a> &#8226;
                        <a class="aWhite" href="contactInfo.html">CONTACT</a>
                    </Div>
                </td>
            </tr>
            <tr class="headerSubTag" height="30">
                <td>
                    <div class="left topBottom">
                        My research interests include Machine Learning, Evolutionary Computation, Artificial Intelligence, and Computer & Network Security. Following are some of the research projects during my PhD study, please refer to <a class="aBlue" target="_self" href="cv.html#publications">HERE</a> for my publication lists. After graduate school, I became more focused on leveraging Machine Learning and Artificial Intelligence to solve interesting business problems, especially in the Customer Experience, Digital, and Data Science domains.
                    </div>
                    <Div class="left dBlue">
                        <b>Selected Research Projects: </b>
                        <A href="#cancerml" class="aBlue" target="_self">Cancer Prediction</A> &#8226;
                        <A href="#securityml" class="aBlue" target="_self">Malware Detection</A> &#8226;
                        <A href="#gart" class="aBlue" target="_self">GART</A> &#8226;
                        <A href="#stock" class="aBlue" target="_self">Stock Prediction</A> &#8226;
                        <A href="#metagenomics" class="aBlue" target="_self">Metagenomics</A>
                    </Div>
                    <hr width=100%>
                </td>
            </tr>
            <tr class="content">
                <td colspan="2">
                    <table cellpadding="0" cellspacing="0" width=100%>
                        <!-- MSTableType="layout" -->
                        <tr>
                            <td width=50% hight=200>
                                <!-- Dissertation -->
                                <div class="center topBottom">
                                    <A name=cancerml class="dBlue tb t12 ">Dissertation: Improving Learning Outcomes by Using Clustering Validity Analysis to Reduce Label Uncertainty</A>
                                </div>
                                <!--
                        <div class="center topBottom"><img src="pic/research/cancer.jpg" /></div>                        
						-->
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, <a class="aBlue" target="_blank" href="http://etal.myweb.uga.edu/">Eric Talevich</a>, <a class="aBlue" target="_blank" href="http://www.bmb.uga.edu/home/people/people.php?fname=natarajan&lname=kannan">Natarajan Kannan</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~khaled/">Khaled Rasheed</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~perdisci/">Roberto Perdisci</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        When people make critical decisions, they often consider the opinions of multiple experts from different domains rather than committing themselves to a single expert or relying on their own judgment. The same principle applies in machine learning for the construction of robust predictive models. However, the outcome of each expert/model might include a certain degree of uncertainty, which affects the robustness of the combined predictive model.
                                        <p>
                                            <p/> In this research, we present two effective machine learning frameworks (ROMP and VAMO) to tackle challenging problems in the domains of cancer prediction and malware clustering. Our frameworks successfully improve the learning outcomes from multiple sources by using cluster validity analysis to reduce label uncertainty.
                                            <p>
                                                <p/> In ROMP, we introduce novel features for identifying cancer-causing mutations in the cancer kinome, and utilize various feature selection methods to evaluate our proposed features. We combine multiple classifiers for the prediction of rare oncogenic mutations, followed by using the Expectation Maximization (EM) clustering algorithm and our self-invented cluster validity metrics to improve the learning outcomes from the ensemble classifier as well as to identify suspicious mutations in the dataset. Our framework successfully discovered rare causative mutations that were later confirmed by lab experiments.
                                                <p>
                                                    <p/> Another framework -- VAMO, provides a fully automated assessment of the quality of malware clustering results. VAMO does not require a manual mapping between malware family labels output by different AV scanners. Furthermore, VAMO does not discard malware samples for which a majority voting-based consensus cannot be reached. Instead, VAMO explicitly deals with the inconsistencies typical of multiple AV labels to build a more representative reference set. Our evaluation, which includes extensive experiments in a controlled setting and a real-world application, show that VAMO performs better than majority voting-based approaches, and provides a way for malware analysts to automatically assess the quality of their malware clustering results.</div>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            <u><b>U, M.</b></u> (2013). <a class="aBlue" target="_blank" href="https://athenaeum.libs.uga.edu/handle/10724/29675">Improving Learning Outcomes by Using Clustering Validity Analysis to Reduce Label Uncertainty</a> (Doctoral Dissertation, The University of Georgia, Athens, GA, USA). Retrieved from <a class="aBlue" target="_blank" href="https://athenaeum.libs.uga.edu/handle/10724/29675">https://athenaeum.libs.uga.edu/handle/10724/29675</a>
                                        </li>
                                    </ul>
                                    <p/>
                                </div>

                                <div class="topBottom">
                                    <hr width=100%>
                                </div>

                                <!-- CML Project-->
                                <div class="center topBottom">
                                    <A name=cancerml class="dBlue tb t12 ">Identification of Cancer-Causing Mutations in the Human Genome</A>
                                </div>
                                <div class="center topBottom"><img src="pic/research/cancer.jpg" /></div>
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, <a class="aBlue" target="_blank" href="http://etal.myweb.uga.edu/">Eric Talevich</a>, <a class="aBlue" target="_blank" href="http://www.bmb.uga.edu/home/people/people.php?fname=natarajan&lname=kannan">Natarajan Kannan</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~khaled/">Khaled Rasheed</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        Cancer is a genetic disease that develops through a series of somatic mutations, a subset of which drive cancer progression. Although cancer genome sequencing studies are beginning to reveal the mutational patterns of oncogenes in various cancers, identifying the small subset of "causative" mutations from the large subset of "non-causative" mutations, which accumulate as a consequence of the disease, is a challenge. In this article, we present an effective machine learning approach for identifying cancer-associated mutations in human protein kinases, a class of signaling proteins known to be frequently mutated in human cancers. We evaluate the performance of 11 well known supervised learners and show that a multiple-classifier approach, which combines the performances of individual learners, significantly improves the classification of known cancer-associated mutations. We introduce several novel features related specifically to structural and functional characteristics of protein kinases and find that the level of conservation of the mutated residue at specific evolutionary depths is an important predictor of oncogenic effect. We consolidate the novel features and the multiple-classifier approach to prioritize and experimentally test a set of rare unconfirmed mutations in the epidermal growth factor receptor tyrosine kinase (EGFR). Our studies identify T725M and L861R as rare cancer-associated mutations inasmuch as these mutations increase EGFR activity in absence of the activating EGF ligand.</div>
                                    <p> </p>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            <u><b>ManChon U</b></u>, Eric Talevich, Khaled Rasheed, Natarajan Kannan; "<a class="aBlue" target="_blank" href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1003545">Prediction and Prioritization of Rare Oncogenic Mutations in the Cancer Kinome Using Novel Features and Multiple Classifiers</a>", <a class="aBlue" target="_blank" href="http://www.ploscompbiol.org/">PLoS Computational Biology</a>, Vol. 10, No. 4, p. e1003545, 2014.
                                        </li>
                                    </ul>
                                    <p/>
                                </div>

                                <div class="topBottom">
                                    <hr width=100%>
                                </div>

                                <div class="center topBottom">
                                    <A name=securityml class="dBlue tb t12">Malware Detection</A>
                                </div>
                                <div class="center topBottom"><img src="pic/research/malware.jpg" /></div>
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~perdisci/">Roberto Perdisci</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        When people make critical decisions, they often consider the opinions of multiple experts rather than committing themselves to a single expert or relying on their own judgment. The same approach should also apply to virus/malware classification/clustering. However, previous studies were mainly focus on simply applying majority voting approach to achieve this goal, while this approach it may result in an overoptimistic estimate of the malware clustering accuracy. In this project, we propose VAMO, a system that provides a fully automated quantitative analysis of the validity of malware clustering results. Unlike previous work, VAMO does not seek a majority voting-based consensus across different AV labels, and does not discard the malware samples for which such a consensus cannot be reached. In essence, VAMO aims to avoid reducing the reference set to only including "easy to cluster" samples. Rather, VAMO explicitly deals with the inconsistencies typical of multiple AV labels to build a more representative reference set. Furthermore, VAMO avoids the need of a (semi-)manual mapping between AV labels from different scanners that was required in previous work. Through an extensive evaluation in a controlled setting and a real-world application, we show that VAMO outperforms majority voting-based approaches, and provides a better way for malware analysts to automatically assess the quality of their malware clustering results.
                                        <p/>
                                    </div>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            Roberto Perdisci and <u><b>ManChon U</b></u>; "VAMO: Towards a Fully Automated Malware Clustering Validity Analysis", <a class="aBlue" target="_blank" href="http://www.acsac.org/">The 28th Annual Computer Security Applications Conference (ACSAC 2012)</a>, Orlando, FL, USA, December, 2012. (Accepted, Acceptance Rate: 19%)
                                        </li>
                                    </ul>
                                    <p/>
                                </div>

                                <div class="topBottom">
                                    <hr width=100%>
                                </div>

                                <div class="center topBottom">
                                    <A name=gart class="dBlue tb t12">Real Time System Scheduler Optimization with Evolutionary Computation Techniques</A>
                                </div>
                                <div class="center topBottom"><img src="pic/research/gart.jpg" /></div>
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~ho/">Chiahsun Ho</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~shelby/">Shelby Funk</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~khaled/">Khaled Rasheed</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        Hard real-time systems require that all jobs are assigned a deadline and the system is deemed to be correct only if all jobs complete execution at or before their deadlines. Such strict timing requirements add to the complexity of the scheduling problem. This complexity is exacerbated when the system is executed on a multiprocessor platform. Even so, scheduling overheads must be kept to a minimum in order for the runtime behavior to be predictable. Thus, real-time scheduling algorithms have the dual requirement of satisfying complex requirements while using fairly simple and straightforward logic. One way an algorithm may achieve this goal is to reduce the overhead due to preemption and migration by rearranging the schedule so as to increase the duration between preemptions. Unfortunately, determining how best to rearrange the jobs is and NP-Complete problem. Hence, we need to use heuristics when scheduling such systems. This leads us to ask a couple of questions. First, what is the best heuristic? Second, is the same heuristic best for all real-time systems?
                                        <p/> We use a Genetic Algorithm to help us answer these questions. Our genetic algorithm based real time system scheduler (GART) is based on the DP-Wrap scheduling algorithm. The genetic algorithm searches through a variety of candidate heuristics to determine the best heuristic for a given task set. Experimental results demonstrate that this approach is able to efficiently identify the best heuristic for all the systems we consider. Moreover, we find that the "best" heuristic does, in fact, depend of various system parameters.
                                    </div>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            <u><b>ManChon U</b></u>, Chiahsun Ho, Shelby Funk and Khaled Rasheed; "GART: A Genetic Algorithm based Real Time System Scheduler", <a class="aBlue" target="_blank" href="http://cec2011.org/">Proceeding of 2011 IEEE Congress on Evolutionary Computation (IEEE CEC 2011)</a>, pp.886-893, New Orleans, LA, USA, 2011.</li>
                                    </ul>
                                    <p/>
                                </div>

                                <div class="topBottom">
                                    <hr width=100%>
                                </div>

                                <div class="center topBottom">
                                    <A name=stock class="dBlue tb t12">Machine Learning Methods for Financial Market Prediction</A>
                                </div>
                                <div class="center topBottom"><img src="pic/research/stock.jpg" /></div>
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~khaled/">Khaled Rasheed</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        Researchers have known for some time that non-linearity exists in the financial markets and that neural networks can be used to forecast market returns. In this project, we implemented a novel stock market prediction system which focuses on forecasting the relative tendency growth between different stocks and indices rather than purely predicting their values. This research utilizes artificial neural network models for estimation. The results are examined for their ability to provide an effective forecast of future values. Certain techniques, such as sliding windows and chaos theory, are employed for data preparation and pre-processing. Our system successfully predicted the relative tendency growth of different stocks with up to 99.01% accuracy.
                                        <p/> Our system can predict the future relative tendency growth among different stocks – or stocks and indices - on a daily bases. Therefore, daily or short term investors and mutual fund managers – especially hedge fund managers - are the most likely group to benefit from our system as they can decide to perform the drop/add of a specific stock on a specific day.
                                    </div>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            <u><b>ManChon U</b></u>, Khaled Rasheed; "A Relative Tendency Based Stock Market Prediction System", <a class="aBlue" target="_blank" href="http://www.icmla-conference.org/icmla10/">Proceeding of The Ninth International Conference on Machine Learning and Applications (ICMLA 2010)</a>, IEEE, pp. 949-953, Washington DC, USA, 2010.
                                        </li>
                                        <br>
                                        <li>
                                            <u><b>ManChon U</b></u>, Zhen Li; "Public Goods Game Simulator with Reinforcement Learning Agents", <a class="aBlue" target="_blank" href="http://www.icmla-conference.org/icmla10/">Proceeding of The Ninth International Conference on Machine Learning and Applications (ICMLA 2010)</a>, IEEE, pp.43-49, Washington DC, USA, 2010.
                                            <a target="_blank" href="doc/2010_ICMLA_PGG.pdf"><img border="0" src="pic/pdf_icon_small.gif"></a>
                                        </li>
                                    </ul>
                                </div>

                                <div class="topBottom">
                                    <hr width=100%>
                                </div>

                                <div class="center topBottom">
                                    <A name=metagenomics class="dBlue tb t12">Scalability and Performance Analysis of Machine Learning Approaches for Gene Classification and Metagenomics</A>
                                </div>
                                <div class="center topBottom"><img src="pic/research/metagenomics.jpg" /></div>
                                <div class="left">
                                    <b>Status:</b> Completed
                                    <p/>
                                    <b>People:</b> <a class="aBlue" target="_blank" href="index.html">ManChon U</a>, Vasim Mahamuda, <a class="aBlue" target="_blank" href="http://www.cs.uga.edu/~khaled/">Khaled Rasheed</a>
                                    <p/>
                                    <b>Introduction:</b>
                                    <div class="leftIndent">
                                        Machine learning algorithms are extensively being used in the field of bioinformatics. With the invent of new genome sequencing techniques there has been a considerable increase in the amount of data available in the biological databases. In research projects, such as determining the structure and function of biological molecules, the role of machine learning is evident. Metagenomics deals with the study of micro-organisms such as prokaryotes that are found in samples from natural environments. The samples obtained from the environment may contain DNA from many different species of micro-organisms including bacteria and archea.
                                        <p/> In this project, we compared the performance of six well known supervised learners - decision trees, naïve bayes, support vector machines, artificial neural networks, bayesian networks and decision tables . We also analyzed the performance of three different meta-learners namely bagging, boosting and stacking. Afterwards, we apply decision trees, bayesian networks and decision tables to see how the performance degrades when the number of species present in the metagenomic sample increases. We also try to see how the performance of the metagenomic sample changes as the percentage of unknown sequences in the metagenomic sample is varied.
                                    </div>
                                    <b>Publication:</b>
                                    <ul>
                                        <li>
                                            <u><b>ManChon U</b></u>, Vasim Mahamuda, Khaled Rasheed; "On the Scalability of Supervised Learners in Metagenomics", <a class="aBlue" target="_blank" href="http://www.icmla-conference.org/icmla10/">Proceeding of The Ninth International Conference on Machine Learning and Applications (ICMLA 2010)</a>, IEEE, pp. 803-807, Washington DC, USA, 2010.
                                            <a target="_blank" href="doc/2010_ICMLA_Metagenomics.pdf"><img border="0" src="pic/pdf_icon_small.gif"></a>
                                        </li>
                                        <br>
                                        <li>
                                            Vasim Mahamuda, <u><b>ManChon U</b></u>, Khaled Rasheed; "Application of Machine Learning Algorithms for Binning Metagenomic Data", <a class="aBlue" target="_blank" href="http://www.world-academy-of-science.org/worldcomp10/ws/conferences/biocomp10">Proceeding of The 2010 International Conference on Bioinformatics & Computational Biology (BIOCOMP'10/ISBN:1-60132-132-5/CSREA)</a>, pp.:68-74, Las Vegas, USA, 2010.
                                            <a target="_blank" href="doc/2010_BIOCOMP_Metagenomics.pdf"><img border="0" src="pic/pdf_icon_small.gif"></a>
                                        </li>
                                    </ul>
                                    <p/>
                                </div>

                            </td>
                        </tr>
                        <tr>
                            <td hight=200>&nbsp;</td>
                        </tr>
                    </table>
                </td>
            </tr>
            <tr class="content">
                <td>
                    <hr>
                </td>
            </tr>
            <tr>
                <td class="footer" colspan="3" height="80" background="pic/footer.gif">
                    <Div class="center vTop">
                        Copyright © 2018. All rights reserved.
                        <br> ManChon (Kevin) U, PhD - CX Expert | Driving Growth Through Digital Transformation & Data Science
                    </Div>
                </td>
            </tr>
        </table>
    </Div>

    <p> </p>
</body>

</html>